templates/strapi/files/README.md

```diff
@@ -0,0 +1,63 @@
+3. This template provides only the API for a full project, but you can modify the repository into a [multi-app project](https://docs.platform.sh/configuration/app/multi-app.html#multiple-applications) where Strapi acts as a separate backend application for the front end of your choice. For example, you can repackage this template into a subdirectory called `backend` to create a project structure that looks like this:
+
+  ```
+  .
+  ├── .git
+  ├── .platform
+  │   ├── routes.yaml
+  │   └── services.yaml
+  ├── backend
+  │   ├── .platform.app.yaml
+  │   ├── api
+  │   ├── config
+  │   ├── extensions
+  │   ├── package.json
+  │   ├── public
+  │   └── yarn.lock
+  └── frontend
+      ├── .platform.app.yaml
+      ├── index.js
+      ├── package-lock.json
+      └── package.json
+  ```
+
+  You can find a full tutorial for how to accomplish this on the [Platform.sh Community Site](https://community.platform.sh/t/use-strapi-as-an-api-backend/435).
+
+## Customizations
+
+The following changes have been made relative to the quickstart Strapi project to run on Platform.sh.  If using this project as a reference for your own existing project, replicate the changes below to your project.
+
+* The `.platform.app.yaml`, `.platform/services.yaml`, and `.platform/routes.yaml` files have been added. These provide Platform.sh-specific configuration and are present in all projects on Platform.sh.  You may customize them as you see fit.
+* An additional module, [config-reader-nodejs](https://github.com/platformsh/config-reader-nodejs), has been added. It provides convenience wrappers for accessing the Platform.sh environment variables.
+* The traditional `config/(development|production|staging)/database.json` file has been replaced with a `database.js` script. This change allows you to leverage `config-reader-nodejs` to dynamically read database (in this case, PostgreSQL) credentials on the fly from the `PLATFORM_RELATIONSHIPS` environment variable, rather than pre-specifying them.
+* The `config/(development|production|staging)/server.json` has been modified to read the `port` configuration from the `PORT` Platform.sh environment variable.
+* The start command is set up to run Strapi as a [`development` environment](https://strapi.io/documentation/3.0.0-beta.x/cli/CLI.html#strapi-develop-dev), allowing you to create new Content Types once it is deployed, even on the `master`/production branch of your project. It calls an added `start.sh` script to run `yarn develop`. When taking your application live, it is recommended that you set Strapi for [`production`](https://strapi.io/documentation/3.0.0-beta.x/cli/CLI.html#strapi-start), where Strapi runs without restarts and the creation of new Content Types is not allowed. `start.sh` includes an example environment-specific start command that can be used to accomplish this using the `PLATFORM_BRANCH` Platform.sh environment variable.
```

Comment:
I didn't really follow this bullet until I read the script.  I think it should be turned around: The start command calls start.sh, which is configured to run dev mode. We recommend adjusting it to automatically run in prod mode on the master branch; see the script for code to enable.  (Or something like that.)